src/UI/Button.elm

```diff
@@ -501,10 +502,10 @@ toggleView cfg size hint toggleMsg current =
             iconLayout size
 
         attrs =
-            [ Primitives.roundedBorders
+            [ Primitives.roundedBorders size
             , paddings
             , borders
-            , Events.onClick <| toggleMsg (not current)
+            , Element.onIndividualClick <| toggleMsg (not current)
```

Comment:
Otherwise clicking on an element over another element triggers their both onClick.
Like the header filter view: Clicking it edits the filter while clicking on the small button over it should only remove the filter.